### DISCUSSION TITLE
Improve the readability of the readme.md

### DISCUSSION THREAD

**Discussion by szaimen** [nextcloud/all-in-one#5362] [Ideas] [answered]
As some people noticed, the readme is not the easiest to read. Would be great if that could be improved.

**Comment by szaimen** [comment]
I've added this to the list in https://github.com/nextcloud/all-in-one/issues/5251 now

**Comment by sunjam** [comment]
I'm open to contributing to this.  Is there actual feedback to reference in regards to this?

**Reply by szaimen** [reply]
Thanks a lot for the offer @sunjam! :)

I was a few months ago talking with someone from the community about this and they suggested to use a template like https://gitlab.com/tgdp/templates/-/blob/main/readme/template_readme.md for example. 
An additional guide can be found here: https://gitlab.com/tgdp/templates/-/blob/main/readme/guide_readme.md

**Comment by sunjam** [comment]
So, is the intention to keep the docs as this singular readme.md file on the github repo, or would it make more sense to add to the actual docs.nextcloud.com as a pull request?  Seems more it would be nice to have the documentation somewhere besides the readme, but I defer to you on what makes sense.

Since only admin, user, dev guides are there... I suppose it isn't the best place either.  I don't know, I see deck has a random readthedocs page and something about all of this documentation all over the place feels ridiculously awkward.  🤷

**Reply by szaimen** [reply]
As the documentation in this repo is indeed mostly about AIO, I would like to keep it here, also to make it easier to work on the docs. ([Aka monorepo](https://en.m.wikipedia.org/wiki/Monorepo)).

**Reply by szaimen** [reply]
But indeed, we could split the readme up into multiple files inside this repo. Posisbly by creating a docs subfolder or comparable

**Comment by punkyard** [comment]
hi, may I join this team?

**Reply by szaimen** [reply]
Yes, of course :)

**Comment by szaimen** [comment, marked as answer]
Just FYI, I just added a TOC to the faq section: https://github.com/nextcloud/all-in-one/pull/6004

**Reply by szaimen** [reply]
I think what would be most interesting is to re-order the sections in the readme and create additonal sub-categories for it. First, restucturing the TOC might be easist and afterwards, adjusting the headings based on it would probably make most sense. See https://github.com/nextcloud/all-in-one?tab=readme-ov-file#faq

**Reply by punkyard** [reply]
I share that idea: making this unique page -that keeps all information about AiO- readable: titles, sub titles, collapsed sections...
I've started this in the first section "presentation". 
but I've received a fail message "Update Psalm baseline: All jobs have failed" i looked online and i just don't get the link between my contribution to a text file, compared to the results i've found searching what this failure relied on. I'm sorry to ask again for help 👻

**Reply by szaimen** [reply]
I just did a restucturing of the whole faq section and added sections: https://github.com/nextcloud/all-in-one/pull/6058
